### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by GodGun968**

### CODE REVIEW

**Review by deirn — APPROVED** [review]

**Comment by GodGun968** [comment]
What does this mean?: head, Attempt #2: All jobs have failed

**Comment by GodGun968** [comment]
Is this my problem?

File: src/resources/resources/assets/waila/lang/zh_cn.json [hunk 1]
```diff
@@ -1,163 +1,227 @@
...
+  "config.waila.plugin_minecraft.attribute.long_armor_max_desc" : "强制数字视图前，图标绘制的最大护甲值",
+
+  "config.waila.plugin_waila"                                   : "Waila核心",
+  "config.waila.plugin_waila.show_mod_name"                     : "显示模组名称",
+  "config.waila.plugin_waila.show_registry"                     : "显示注册名称",
+  "config.waila.plugin_waila.show_blocks"                       : "显示方块",
+  "config.waila.plugin_waila.show_fluids"                       : "显示流体",
+  "config.waila.plugin_waila.show_fluids_desc"                  : "尝试显示流体",
+  "config.waila.plugin_waila.show_entities"                     : "显示实体",
+  "config.waila.plugin_waila.show_icon"                         : "显示图标"
```
Comment (deirn) [inline src/resources/resources/assets/waila/lang/zh_cn.json]: Missing comma :P
```suggestion
  "config.waila.plugin_waila.show_icon"                         : "显示图标",
```

**Comment by GodGun968** [comment]
oh ,sorry

**Comment by GodGun968** [comment]
Now everything is ready :)

**Comment by deirn** [comment]
Thanks!

**Comment by GodGun968** [comment]
Hello, do you want me to do Chinese translation for your project all the time?

**Pull request merged**